.changeset/flat-seahorses-collect.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
I don't think these are all minor:
* freestyle - shouldn't be mentioned at all, it is just patch due to a change in ui5-config that will be automatically generated during the release
* ui5-app - it is just a path, projects that worked before still work, just a little bit better performance
* elements - minor is correct here
* fpm - just a patch in an incorrect validate function
* preview-middleware - just a patch that the consumer won't even notice